### PR TITLE
fix(menus): macOS Services menu not working, closes #243

### LINKED ISSUE #243: `Services` menu item not working
The `Services` menu item should have a submenu, but it's just a normal menu item that doesn't do anything:
x

### PR DESCRIPTION
**Pull request by caesar**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### Other information
closes #243

### CODE REVIEW

**Review by amrbashir — APPROVED** [review]

**Review by amrbashir — CHANGES_REQUESTED** [review]
missing changefile

**Comment by caesar** [comment]
> missing changefile

Wait, there are changefiles now? Nice!

**Review by amrbashir — APPROVED** [review]

**Comment by amrbashir** [comment]
thank you

**Comment by wanghui2021** [comment]
image

Is this divider redundant?

File: src/platform_impl/macos/menu.rs [hunk 1]
```diff
@@ -308,10 +308,12 @@ impl Menu {
       )),
       MenuItem::Services => unsafe {
         let item = make_menu_item("Services", None, None, MenuType::MenuBar);
+        // we have to assign an empty menu as the app's services menu, and macOS will populate it
+        let services_menu = NSMenu::alloc(nil).autorelease();
```
Comment (caesar) [inline src/platform_impl/macos/menu.rs:312]:
```suggestion
        let services_menu = NSMenu::new(nil).autorelease();
```

**Review by caesar — COMMENTED** [review]
> Is this divider redundant?

It is indeed redundant. When I implemented this I wasn't sure how to solve it, but I have since found the solution, which is simply to change `NSMenu::alloc(nil)` to `NSMenu::new(nil)` as shown in the code review below.

File: src/platform_impl/macos/menu.rs (hunk 1 above)
Comment (wanghui2021) [reply src/platform_impl/macos/menu.rs:312]: when will this be fixed？

**Pull request merged**